app/src/main/java/com/github/polypoly/app/menu/JoinGameLobbyActivity.kt

```diff
@@ -32,25 +33,33 @@ import com.github.polypoly.app.game.GameLobby
 import com.github.polypoly.app.game.user.Skin
 import com.github.polypoly.app.game.user.Stats
 import com.github.polypoly.app.game.user.User
+import com.github.polypoly.app.global.GlobalInstances.Companion.remoteDB
 import com.github.polypoly.app.global.Settings.Companion.DB_GAME_LOBIES_PATH
-import com.github.polypoly.app.network.FakeRemoteStorage
 import com.github.polypoly.app.network.getAllValues
 import com.github.polypoly.app.network.getValue
 import com.github.polypoly.app.ui.theme.PolypolyTheme
 import com.github.polypoly.app.ui.theme.UIElements
 import kotlinx.coroutines.delay
 import timber.log.Timber
+import java.util.concurrent.CompletableFuture
 
 /**
  * Activity where the user can join a gameLobby
  */
 class JoinGameLobbyActivity : MenuActivity("Join Game") {
+    companion object {
+        const val POLLING_INTERVAL = 5000L
```

Comment:
Maxime said in #77 that this will be changed, because polling is worse than listening, do you agree?

Reply:
Yes I also added a TODO comment line 213 (where it is used) as it is only the intermediate implementation until event listening is implemented with firebase.